x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -323,14 +345,16 @@ export class AlertsClient {
    */
   private async mgetAlertsAuditOperate({
     ids,
-    status,
     indexName,
     operation,
+    fieldToUpdate,
+    validate,
   }: {
     ids: string[];
-    status: STATUS_VALUES;
     indexName: string;
     operation: ReadOperations.Find | ReadOperations.Get | WriteOperations.Update;
+    fieldToUpdate: (source: ParsedTechnicalFields | undefined) => Record<string, unknown>;
+    validate?: (source: ParsedTechnicalFields | undefined) => void;
```

Comment:
To support cases making a single call instead of pmap we'd need to change this method to combining the `ids` and `indexName` to something like:

```
{
...
  alerts: Array<{ ids: string[]; indexName: string; }>,
...
}
```

And then the `this.esClient.mget<ParsedTechnicalFields>` call can put everything in the `body` per the docs here: https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-multi-get.html